lapis/simulator.py

```diff
@@ -26,6 +27,8 @@ def __init__(self, seed=1234):
         random.seed(seed)
         self.job_queue = Queue()
         self.pools = []
+        self.storage_list = []
+        self.fileprovider = FileProvider()
```

Comment:
Why do you have a `storage_list` next to `fileprovider`? The `fileprovider`itself takes also care on managing the list of storages, doesn't it?

Reply:
You're right. This is a remnant of a former approach.